test/rocc/util.h

```diff
@@ -0,0 +1,135 @@
+// See LICENSE for license details.
```

Comment:
Is any of the code in this header actually used? It's included in syscalls.c, but none of the code here appears to be used anywhere else.

Reply:
static_assert is used somewhere. The binary fails to link without including util.h due to a missing "static_assert".